index.html

```diff
@@ -6,25 +6,27 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <script src="script.js" defer></script>
+  <script src="projects.js" defer></script>
+  <!-- <script src="form.js" defer></script> -->
```

Comment:
- [ ] please remove the unused uncommented code in order to keep  you file clean and pro